packages/uswds-core/src/styles/settings/_settings-components.scss

```diff
@@ -110,6 +110,15 @@ $theme-identifier-max-width: "desktop" !default;
 $theme-identifier-primary-link-color: default !default;
 $theme-identifier-secondary-link-color: "base-light" !default;
 
+// In-Page Navigation
+$theme-in-page-nav-background-color: "white" !default;
```

Comment:
Thinking it might be helpful to define this with the theme body background color setting. It feels like it could be the most common use pattern.

```
@use "settings-color" as color;

...

$theme-in-page-nav-background-color: color.$theme-body-background-color !default;
```

Reply:
I think I'll set it as `default` as in a component like Breadcrumb. That way it will pick up `$theme-body-background-color`

Reply:
And that's what I did!